deploy/hco.cr.yaml

```diff
@@ -3,4 +3,10 @@ apiVersion: hco.kubevirt.io/v1beta1
 kind: HyperConverged
 metadata:
   name: kubevirt-hyperconverged
-spec: {}
+spec:
+  hyperConvergedConfig:
+    placement:
+      infra:
+        affinity: {}
```

Comment:
Where are tolerations?

Reply:
The right question is why this is here: this is just the example CR, not the CRD where we have:
```
	// +optional
	Affinity corev1.Affinity `json:"affinity,omitempty"`
```
So I don't expect it here if empty

Reply:
It disappeared removing one nested level